packages/components/atoms/f-toggle/src/components/Toggle.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+    <label
+        :for="uniqueId"
+        data-test-id="toggle-component"
+        :class="[$style['c-toggle'],
+                 {
+                     [$style['c-toggle--checked']]: isChecked,
+                     [$style['c-toggle--disabled']]: $attrs.disabled,
+                     [$style['c-toggle--modeRTL']]: isModeRightToLeft
+                 }]">
+        <input
+            :id="uniqueId"
+            role="switch"
+            type="checkbox"
+            v-bind="$attrs"
+            :class="$style['c-toggle-input']"
+            :aria-checked="isChecked"
+            :aria-labelledby="ariaLabelledBy"
+            :aria-describedby="ariaDescribedBy"
```

Comment:
More of a question rather than a request to change it – as you're binding `$attrs` to the input, if you set an `aria-labelledby` and  `aria-describedby` on the component, wouldn't those automatically get passed down and applied without using the prop?

I do like the way the props encourage the use of aria attributes, but just wondering if we're adding props that aren't needed